src/Constants.js

```diff
@@ -46,8 +46,8 @@ exports.GiveawayMessages = {
  * @typedef MessageObject
  *
  * @property {string} [content] The raw message
- * @property {Discord.MessageEmbed|Discord.MessageEmbedOptions} [embed] The embed
- * @property {Array<Discord.MessageActionRow|Discord.MessageActionRowOptions>} [components] The components.<br>"content" or "embed" is required to be defined.
+ * @property {Discord.EmbedBuilder|Discord.EmbedData|Discord.APIEmbed} [embed] The embed
+ * @property {Array<Discord.ActionRowBuilder<Discord.MessageActionRowComponentBuilder>|Discord.MessageActionRowComponentData>} [components] The components.<br>"content" or "embed" is required to be defined.
```

Comment:
the types should be https://discord.js.org/#/docs/discord.js/main/class/ActionRowBuilder?scrollTo=s-from now, no?